tests/features/collection/notification.collection.feature

```diff
@@ -219,6 +253,3 @@ Feature: Notification test for the collection transitions.
       | recipient | NC member2                                                                  |
       | subject   | The collection NC to delete was deleted.                                    |
       | body      | The collection "NC to delete", of which you are a member, has been deleted. |
-
```

Comment:
How come this was removed from here and moved on top? Just out of curiosity.

Reply:
This does not actually make any difference when the tests are green. However, when I was changing the emails, I had to re run the tests multiple times. After fixing the email during the manual proposal of the collection, other cases were breaking. However, the step is not an after scenario step so it was never being run. Then I had to clean the collection manually to re run it.

Like this, the collection is cleaned up right after the relevant lines are ran

Reply:
Oh yes perfect, it's actually an improvement then. 👍